src/render.ts

```diff
@@ -45,6 +45,14 @@ const controls = {
   ArrowDown: {
     t: EventType.Drop,
   },
+  Space: [
```

Comment:
this won't "freeze" the omino, which could lead to weird behavior if, eg, you try to drop your new omino on top of the previous one

Reply:
never mind.